Mathlib/Analysis/InnerProductSpace/PiL2.lean

```diff
@@ -679,8 +674,7 @@ theorem Complex.map_isometryOfOrthonormal (v : OrthonormalBasis (Fin 2) ℝ F) (
     Complex.isometryOfOrthonormal (v.map f) = (Complex.isometryOfOrthonormal v).trans f := by
   simp only [isometryOfOrthonormal, OrthonormalBasis.map, LinearIsometryEquiv.symm_trans,
     LinearIsometryEquiv.symm_symm]
-  -- Porting note: `LinearIsometryEquiv.trans_assoc` doesn't trigger in the `simp` above
-  rw [LinearIsometryEquiv.trans_assoc]
+  rfl
```

Comment:
I think this is a step backwards. The `simp` application is still broken.

Reply:
reverted this. 

(Not sure what the real fix here is. The entire proof, even the `simp only` in the line before can be done by `by rfl`. It does help to make `LinearIsometryEquiv.trans_assoc` a `simp` lemma).